source/catalog.html

```diff
@@ -0,0 +1,254 @@
+    <a class="page-header__logo">
+      <img class="page-header__logo-image" src="" width="33" height="38" alt="Кэт Энерджи">
+      <img class="page-header__logo-text" src="" width="101" height="18" alt="">
+    </a>
+    <nav class="main-nav">
+      <button class="main-nav__toggle" type="button">
+        <span class="visually-hidden">Открыть меню</span>
+      </button>
+      <div class="main-nav__wrapper">
+        <ul class="main-nav__list site-list">
+          <li class="site-list__item site-list__item--active">
+            <a class="site-list__link" href="index.html">Главная</a>
+          </li>
+          <li class="site-list__item">
+            <a class="site-list__link" href="catalog.html">Каталог продукции</a>
+          </li>
+          <li class="site-list__item">
+            <a class="site-list__link" href="form.html">Подбор программы</a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+
+  <main class="page-main">
+    <section class="catalog">
+      <h1 class="catalog__title">Каталог продукции</h1>
+      <div class="product">
+        <h2 class="visually-hidden">Карточки товара</h2>
+        <ul class="product__list">
+          <li class="product__card">
+            <a class="product__image-link" href="#">
+              <img class="product__image" src="" width="68" height="86" alt="500gr">
+            </a>
+            <a class="product__title" href="#">
+              <h3 class="product__card-title">Cat Energy PRO 500 г</h3>
+            </a>
+            <d1 class="product__specifications">
+              <dt class="product__specifications-definition">Масса</dt>
+              <dd class="product__specifications-meaning">500 г</dd>
+              <dt class="product__specifications-definition">Вкус</dt>
+              <dd class="product__specifications-meaning">Курица</dd>
+              <dt class="product__specifications-definition">Цена</dt>
+              <dd class="product__specifications-meaning">700 Р.</dd>
+              <a href="#" class="button product__button">заказать</a>
+          </li>
+          <li class="product__card">
+            <a class="product__image-link" href="#">
+              <img class="product__image" src="" width="84" height="101" alt="1000gr">
+            </a>
+            <a class="product__title" href="#">
+              <h3 class="product__card-title">Cat Energy PRO 1000 г</h3>
+            </a>
+            <d1 class="product__specifications">
+              <dt class="product__specifications-definition">Масса</dt>
+              <dd class="product__specifications-meaning">1000 г</dd>
+              <dt class="product__specifications-definition">Вкус</dt>
+              <dd class="product__specifications-meaning">Курица</dd>
+              <dt class="product__specifications-definition">Цена</dt>
+              <dd class="product__specifications-meaning">1000 Р.</dd>
+              <a href="#" class="button product__button">заказать</a>
```

Comment:
Лучше писать в нормальном регистре
```suggestion
              <a href="#" class="button product__button">Заказать</a>
```

Reply:
Поняла, исправила